index.html

```diff
@@ -38,5 +38,95 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
+            <div class="card-1-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-2">
+            <div class="card-2-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-3">
+            <div class="card-3-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-4">
+            <div class="card-4-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="ams">
+            <h1>About Myself</h1>
+            <p>
+               Hello I'm a software developer! I can help you build a product, feature or website
+               Look through some of my work and experiance! If you like what you see and have a project you need coded,
+               don't hestiate to contact me.
+            </p>
+            <div class="letconnect_text">LET'S CONNECT</div>
```

Comment:
- [ ] As required for the project, kindly make sure the `LET'S CONNECT` text use a header tag `hx(h3,...)` as it is a title. And if possible you can add inside a `<a>` tag as it can lead users to your social account to coonect with you such as `Linkedin`.
```html
<h3 class="letconnect_text"><a href="https://www....">LET'S CONNECT</a></h3>